### DISCUSSION TITLE
single file alternative to OIDCMetadataDir?

### DISCUSSION THREAD

**Discussion by dnmvisser** [OpenIDC/mod_auth_openidc#1222] [Q&A] [answered]
Hi, we're using a setup with many different OIDC providers, for which we use `OIDCMetadataDir`, as per [docs](https://github.com/OpenIDC/mod_auth_openidc/wiki/Multiple-Providers). It works, but this file system based approach means there are many files, which adds some overhead here (for context: we're using ansible to create the templates).
Also it means I have to manage the clean up process for files that should not be there.

Is there perhaps an alternative configuration method, whereby everything is configured in a single file? That would solve both of my problems.

**Comment by zandbelt** [comment, marked as answer]
no, there is no such alternative configuration method
